docs/installation.rst

```diff
@@ -52,20 +52,14 @@ Execute the following command to install the latest DALI for specified CUDA vers
 
 * for CUDA 11.0:
 
-.. code-block:: bash
-
-   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda110
-
-* for CUDA 11.1:
-
   .. note::
-    CUDA 11.1 build uses CUDA toolkit enhanced compatibility. It is built with the latest CUDA 11.x
-    toolkit while it can run on CUDA 11.0 capable drivers 450.80 or later. Using the newest driver may enable
-    additional functionalities.
+    CUDA 11.0 build uses CUDA toolkit enhanced compatibility. It is built with the latest CUDA 11.x
+    toolkit while it can run on the latest, stable CUDA 11.0 capable drivers (450.80 or later).
+    Using the newest driver (455.x, for example) may enable additional functionalities.
```

Comment:
I don't think that we need to put an "example" of latest driver - that's bound to go stale.
```suggestion
    Using the latest driver may enable additional functionality.
```

Reply:
Done